.golangci.yml

```diff
@@ -0,0 +1,61 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# GOLANG LINTER RULES
+# ----------------------------------------------------------------------------------------------------------------------
+
+# configure golangci-lint
+# see https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml
+issues:
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - dupl
+        - gosec
+        - goconst
+linters:
+  enable:
+#    - bodyclose
+#    - deadcode
+#    - depguard
+#    - dupl
+#    - errcheck
+#    - gocritic
+#    - gofmt
+#    - goconst
+#    - goimports
```

Comment:
shouldn't this config be in the pre-commit hooks... to be aligned over all repos and not different in every repo? @soerenmartius

Reply:
Nope. We just only define the hook inside the [pre-commit-hooks repository](https://github.com/mineiros-io/pre-commit-hooks). The dependency ( golangci-lint) is installed as a dependency inside the [build-tools](https://github.com/mineiros-io/build-tools) container. golangci-lint can optionally load a config `.golangci.yml`. The config though belongs to the target repository since every repository could have different linter requirements. 

Does that make sense to you?

Reply:
Not really as we want to assure the same code guidelines over all repositories.. So we would end up copying the same config everywhere? So it could be active by default..?